vj4/server.py

```diff
@@ -10,7 +10,7 @@
 from vj4 import app
 from vj4.util import options
 
-options.define('listen', default='http://127.0.0.1:8888', help='Server listening address.')
```

Comment:
because it acts like a web server and it should follow what most web servers do: listen `0.0.0.0`.